lib/service_actor.rb

```diff
@@ -80,6 +115,9 @@ def run
 
   # Can be called from inside an actor to stop execution and mark as failed.
   def fail!(**arguments)
+    # Should it be context (without the `@`)? I know it makes virtually no
+    # difference but it seems more sense regarding the `Attributable` module
+    # that is overriding the `#context` method.
```

Comment:
Yeah, could be. I'll change it, since I'd rather be using methods rather than instance variables when I can.